expression/builtin_convert_charset.go

```diff
@@ -256,26 +256,97 @@ func BuildFromBinaryFunction(ctx sessionctx.Context, expr Expression, tp *types.
 	return FoldConstant(res)
 }
 
+type funcProp int8
+
+const (
+	funcPropNone funcProp = iota
```

Comment:
if the `funcPropNone` means do nothing, why not remove it.

Reply:
These functions are listed there to show that they have been considered.